django/conf/locale/dsb/LC_MESSAGES/django.po

```diff
@@ -6,10 +6,10 @@ msgid ""
 msgstr ""
 "Project-Id-Version: django\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2023-01-17 02:13-0600\n"
-"PO-Revision-Date: 2023-04-25 06:49+0000\n"
+"POT-Creation-Date: 2023-09-18 11:41-0300\n"
+"PO-Revision-Date: 2013-04-25 06:49+0000\n"
```

Comment:
`PO-Revision-Date` must be manually bumped to be later than `POT-Creation-Date`.

Reply:
On release day, will run:
```
find django -name '*.po' | xargs sed -ri "s/PO-Revision-Date: [0-9\-]+ /PO-Revision-Date: $(date -I) /g"
```